avocado.spec

```diff
@@ -47,11 +47,12 @@ these days a framework) to perform automated testing.
 %{__install} -m 0644 man/avocado.1 %{buildroot}%{_mandir}/man1/avocado.1
 %{__install} -m 0644 man/avocado-rest-client.1 %{buildroot}%{_mandir}/man1/avocado-rest-client.1
 
-# Running the unittests is currently disabled on EL6 because fabric is
-# missing on EPEL 6 and also on EL7 because python-flexmock is missing
-# on EPEL7.
-%if !0%{?rhel}
+# Running the unittests is currently disabled on all platforms but
+# Fedora 25.  EL6 because fabric is missing on EPEL 6, on EL7 because
+# python-flexmock is missing on EPEL7 and on Fedora 24 because of
+# errors related to the fabric version shipped there.
 %check
+%if 0%{?fedora} == 25
```

Comment:
Well in theory this is true for all supported versions by us, but wouldn't it be possible to use `blacklist` rather than `whitelist`? Someone might want to build it on F23 or F26 and the selftests should run in such case...